### PR TITLE
Remove please from QR code

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Per MS style guide:
Avoid please except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra please review

File: src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml
Comment (tdykstra) [inline src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: For accessibility, link text "documentation" is not very descriptive; you could rearrange the text:
```suggestion
            <div class="alert alert-info">Learn how to <a href="https://go.microsoft.com/fwlink/?Linkid=852423">enable QR code generation</a>.</div>
```

File: src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Manage/EnableAuthenticator.cshtml
Comment (tdykstra) [inline src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Manage/EnableAuthenticator.cshtml]:
```suggestion
            <div class="alert alert-info">Learn how to <a href="https://go.microsoft.com/fwlink/?Linkid=852423">enable QR code generation</a>.</div>
```

**Review by tdykstra — APPROVED** [review]

File: src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml
Comment (pranavkm) [inline src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: /cc @blowdart

Comment (blowdart) [reply src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: RUDENESS

**Comment by Rick-Anderson** [comment]
Same change in https://github.com/aspnet/Scaffolding/pulls

File: src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml
Comment (Rick-Anderson) [reply src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: @blowdart  please suggest alternate wording. How about

To enable QR, hit up blowdart on twitter.

Comment (blowdart) [reply src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: If we're not allowed to be polite and say Please then fine, whatever

**Comment by Rick-Anderson** [comment]
@blowdart  can you **please** grudgingly sign off on https://github.com/aspnet/Scaffolding/pull/1027

**Comment by anurse** [comment]
> @tdykstra please review

I think you mean "~~please~~ review" :trollface:

LGTM, but I suppose @blowdart or @HaoK should confirm in case there are other places this might appear?

**Comment by Rick-Anderson** [comment]
@HaoK  said just here and https://github.com/aspnet/Scaffolding/pulls

**Review by vijayrkn — APPROVED** [review]

File: src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml
Comment (tdykstra) [reply src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: For accessibility, link text is supposed to be more descriptive than "click here".

Comment (Rick-Anderson) [reply src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Manage/EnableAuthenticator.cshtml]: Additionally, less is more.

**Review by mkArtakMSFT — APPROVED** [review]
Please wait for @blowdart to sign off before merging this.

**Comment by Rick-Anderson** [comment]
@blowdart  did sign off, but his comments are hidden or deleted. He preferred please, but said said fine.

See https://github.com/aspnet/Scaffolding/pull/1027, which is exactly the same code and changes.

**Comment by blowdart** [comment]
What he said :) Merge away

**Pull request merged**
